qiime2/sdk/plugin_manager.py

```diff
@@ -93,18 +93,38 @@ def _integrate_plugin(self, plugin):
     # databases rollback/commit
 
     @property
+    def importable_formats(self):
+        """Return set of formats that are importable.
```

Comment:
The return type we're looking for here is a `dict` mapping format names (strings) to `FormatRecord` objects (it's the same structure that `PluginManager.formats` uses). It looks like a `set` of `FormatRecord` objects are being returned currently.